src/iosrv/srv.pb-c.c

```diff
@@ -0,0 +1,105 @@
+/* Generated by the protocol buffer compiler.  DO NOT EDIT! */
+/* Generated from: srv.proto */
+
+/* Do not generate deprecated warnings for self */
+#ifndef PROTOBUF_C__NO_DEPRECATED
+#define PROTOBUF_C__NO_DEPRECATED
+#endif
+
+#include "srv.pb-c.h"
+void   srv__notify_ready_req__init
+                     (Srv__NotifyReadyReq         *message)
+{
+  static const Srv__NotifyReadyReq init_value = SRV__NOTIFY_READY_REQ__INIT;
+  *message = init_value;
+}
+size_t srv__notify_ready_req__get_packed_size
+                     (const Srv__NotifyReadyReq *message)
+{
+  assert(message->base.descriptor == &srv__notify_ready_req__descriptor);
+  return protobuf_c_message_get_packed_size ((const ProtobufCMessage*)(message));
+}
+size_t srv__notify_ready_req__pack
+                     (const Srv__NotifyReadyReq *message,
+                      uint8_t       *out)
+{
+  assert(message->base.descriptor == &srv__notify_ready_req__descriptor);
+  return protobuf_c_message_pack ((const ProtobufCMessage*)message, out);
+}
+size_t srv__notify_ready_req__pack_to_buffer
+                     (const Srv__NotifyReadyReq *message,
+                      ProtobufCBuffer *buffer)
+{
+  assert(message->base.descriptor == &srv__notify_ready_req__descriptor);
+  return protobuf_c_message_pack_to_buffer ((const ProtobufCMessage*)message, buffer);
+}
+Srv__NotifyReadyReq *
+       srv__notify_ready_req__unpack
+                     (ProtobufCAllocator  *allocator,
+                      size_t               len,
+                      const uint8_t       *data)
+{
+  return (Srv__NotifyReadyReq *)
+     protobuf_c_message_unpack (&srv__notify_ready_req__descriptor,
+                                allocator, len, data);
+}
+void   srv__notify_ready_req__free_unpacked
+                     (Srv__NotifyReadyReq *message,
+                      ProtobufCAllocator *allocator)
+{
+  if(!message)
+    return;
+  assert(message->base.descriptor == &srv__notify_ready_req__descriptor);
+  protobuf_c_message_free_unpacked ((ProtobufCMessage*)message, allocator);
```

Comment:
(style)  "(foo*)" should be "(foo *)"

(style)  space prohibited between function name and open parenthesis '('